runtime.md

```diff
@@ -129,18 +129,19 @@ This operation MUST run the user-specified program as specified by [`process`](c
 This operation MUST generate an error if `process` was not set.
 
 ### <a name="runtimeKill" />Kill
-`kill <container-id> <signal>`
+`kill <container-id> [-a,--all] <signal>`
```

Comment:
Duplicate of 
- https://github.com/opencontainers/runtime-spec/pull/1190

Reply:
https://github.com/opencontainers/runtime-spec/pull/1190#pullrequestreview-1350798670

Reply:
OK, I changed this PR to `clarify kill and delete operation for shared pid namespace container`, let's discuss `-a` in #1190 .